src/GitHub.Exports.Reactive/Extensions/ConnectionManagerExtensions.cs

```diff
@@ -1,80 +1,20 @@
 ﻿using System;
-using System.Linq;
-using System.Reactive.Linq;
-using System.Reactive.Threading.Tasks;
 using System.Threading.Tasks;
 using GitHub.Factories;
 using GitHub.Models;
-using GitHub.Primitives;
 using GitHub.Services;
 
 namespace GitHub.Extensions
 {
     public static class ConnectionManagerExtensions
     {
-        public static IObservable<bool> IsLoggedIn(this IConnectionManager cm)
-        {
-            Guard.ArgumentNotNull(cm, nameof(cm));
-
-            return Observable.FromAsync(async () =>
-            {
-                var connections = await cm.GetLoadedConnections();
-                return connections.Any(x => x.ConnectionError == null);
-            });
-        }
-
-        public static IObservable<bool> IsLoggedIn(this IConnectionManager cm, HostAddress address)
-        {
-            Guard.ArgumentNotNull(cm, nameof(cm));
-            Guard.ArgumentNotNull(address, nameof(address));
-
-            return Observable.FromAsync(async () =>
-            {
-                var connections = await cm.GetLoadedConnections();
-                return connections.Any(x => x.HostAddress == address && x.ConnectionError == null);
-            });
-        }
-
-        public static IObservable<bool> IsLoggedIn(this IConnection connection)
-        {
-            Guard.ArgumentNotNull(connection, nameof(connection));
-
-            return Observable.Return(connection?.IsLoggedIn ?? false);
-        }
-
-        public static IObservable<IConnection> GetConnection(this IConnectionManager cm, HostAddress address)
-        {
-            Guard.ArgumentNotNull(cm, nameof(cm));
-            Guard.ArgumentNotNull(address, nameof(address));
-
-            return cm.GetLoadedConnections()
-                .ToObservable()
-                .Select(x => x.FirstOrDefault(y => y.HostAddress == address));
-        }
-
-        public static IObservable<IConnection> GetLoggedInConnections(this IConnectionManager cm)
-        {
-            Guard.ArgumentNotNull(cm, nameof(cm));
-
-            return cm.GetLoadedConnections()
-                .ToObservable()
-                .Select(x => x.FirstOrDefault(y => y.IsLoggedIn));
-        }
-
         public static async Task<IModelService> GetModelService(
```

Comment:
Should this method be in `GitHub.Exports.Reactive`?  I didn't notice any Rx stuff that it depends on.

Reply:
`IModelService` is reactive so that's why it's still here.